modules/installation-bootstrap-gather.adoc

```diff
@@ -49,15 +49,20 @@ command:
 ----
 $ ./openshift-install gather bootstrap --dir=<directory> \ <1>
     --bootstrap <bootstrap_address> \ <2>
-    --master "<master_address> <master_address> <master_address>" <3>
+    --master <master_1_address> \ <3>
+    --master <master_2_address> \ <3>
+    --master <master_3_address>" <3>
 ----
 <1> `installation_directory` is the directory you stored the {product-title}
 definition files that the installation program creates.
 <2> `<bootstrap_address>` is the fully-qualified domain name or IP address of
 the cluster's bootstrap machine.
-<3> `<master_address>` is the fully-qualified domain name or IP address of a
-control plane, or master, machine in your cluster. Specify a space-delimited
-list that contains all the control plane machines in your cluster.
+<3> For each control plane, or master, machine in your cluster, replace `<master_*_address>` with its fully-qualified domain name or IP address.
++
+[NOTE]
+====
+A default cluster contains three control plane machines. List all of your control plane machines, even if you aren't using three like the example above.
```

Comment:
I think the "must" could have stayed.

```suggestion
A default cluster contains three control plane machines. List all of your control plane machines as shown, no matter how many your cluster uses.
```

Our style guide prohibits both contractions and the use of "above" like that.

Reply:
Sounds good :)  I can squash back down to one commit when I get back to my laptop.

Reply:
Great! Thank you, Trevor!

Reply:
Squashed and rebased onto the current master with 8a32a0925 -> 1e49f6876.